modules/apps/portal/portal-html-parser-impl/src/test/java/com/liferay/portal/html/parser/internal/HtmlParserImplTest.java

```diff
@@ -40,6 +45,25 @@ public void testExtraction() {
 			"onclick removal",
 			_htmlParserImpl.extractText(
 				"<div onclick=\"honk()\">onclick removal</div>"));
+
+		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
+				"com.liferay.portal.html.parser.internal.HtmlParserImpl",
+				LoggerTestUtil.ERROR)) {
+
+			_htmlParserImpl.extractText(
+				"<div name=\"test\" class=\"test\">Okay</div>");
```

Comment:
As we talked, we'd like to assert that the result of rendering our clay tags is ok for the parser. A possibility to do this is to inject here the output of rendering the `frontend-taglib-clay-sample-web` widget. This is how portlets are rendered in a fragment → [https://github.com/liferay/liferay-portal/blob/8a6cd7d51d97dd4decfe345abe4d141073c[…]ray/fragment/internal/renderer/FragmentPortletRendererImpl.java](https://github.com/liferay/liferay-portal/blob/8a6cd7d51d97dd4decfe345abe4d141073c7a976/modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/FragmentPortletRendererImpl.java) Looks an interesting place to look at.